clients/admin-ui/src/mocks/data.ts

```diff
@@ -303,3 +303,27 @@ export const mockConsentableItems: ConsentableItem[] = [
     unmapped: true,
   },
 ];
+
+// Mock data for Okta applications
+export const MOCK_OKTA_APPS = [
+  {
+    urn: "urn:okta:app:12345678-1234-1234-1234-123456789012",
+    name: "Salesforce",
+    description: "Customer relationship management platform",
+    resource_type: "okta_app",
```

Comment:
```suggestion
    resource_type: "Okta App",
```
to match backend stubs